### DISCUSSION TITLE
artist website - I ditched SquareSpace for NextJs

### DISCUSSION THREAD

**Discussion by haschdl** [vercel/next.js#42600] [Show and tell]
A friend of mine introduced me to NextJs a few months back and I decided it could be my way out of Squarespace - a great product but too expensive as I don't make any income from my website. 

It's a basic portfolio website, and I wanted to keep it simple. For example there is no e-commerce feature - the "add to cart" button redirects to the contact form.

It would be great to hear some feedback on issues that are common on first time projects like this. 
Address: [hscheidl.art](https://hscheidl.art)

**Comment by GabenGar** [comment]
That hover effect should be a transition with duration, otherwise the scaling jump is too jarring.
The entire card being a link brings a lot of accessibility issues in general, one of them is not being able to select the text underneath. The other one is the hover effect doesn't trigger while browsing them with `Tab` key (because it's not a hover).
The image component should have some sort of transparent background/outline, if only to separate it from the background.
Otherwise you get spills like this:
![H Scheidl Home](https://user-images.githubusercontent.com/87906913/209814851-ec5a4726-b68e-4c8f-a4d8-638ec687890e.png)
